gfx/Mushroom-Dream/pngs_tiny_16x16/weather/rain.json

```
@@ -0,0 +1,4 @@
+{
+	"id": "weather_rain_drop",
+	"bg": ["rain", "rain2"]
```

```suggestion
  "bg": [
    { "weight": 1, "sprite": "rain" },
    { "weight": 1, "sprite": "rain2" }
  ]
```
Are you sure you don't need something like this in the json?